go/vt/vttablet/tabletserver/queryz.go

```diff
@@ -89,6 +89,14 @@ func (qzs *queryzRow) timePQ() float64 {
 	return float64(qzs.tm) / (1e9 * float64(qzs.Count))
 }
 
+func (qzs *queryzRow) rowsPQ() float64 {
```

Comment:
nit: may be more readable to call this `rowsPerQuery` and change all other functions similarly.